english/warning/heed.md

```diff
@@ -0,0 +1,26 @@
+"So what is the problem?" you ask the man with the crazy hair.
+
+The old man replies, "Within this area there exists a bridge. A bridge to multiple universes."
+
+As you start to walk away, he continues, "Don't walk away now, I haven't finished."
+
+"The fabric that keeps these mutliverses from breaking into our reality is thin here. You will
+find strange happenings going on. Some of these incursions will be harmless, others can be deadly.", the old man continues.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.